eos/utils/qcd.cc

```diff
@@ -99,14 +99,26 @@ namespace eos
     }};
 
     double
-    QCD::alpha_s(const double & mu, const double & alpha_s_0, const double & mu_0, const BetaFunction & beta)
+    QCD::alpha_s(const double & mu, const double & alpha_s_0, const double & mu_0, const BetaFunction & beta, unsigned int loop_order)
     {
+        if (loop_order == 0)
+        {
+            throw InternalError("QCD::alpha_s: zero loop order not implemented");
+        }
+        if (loop_order > 4)
+        {
+            throw InternalError("QCD::alpha_s: loop order " + stringify(loop_order) + " not yet implemented");
+        }
+
         double a = alpha_s_0 / M_PI;
+        std::array<int, 4> switches;
```

Comment:
I would turn the switches into ``std::array<double, 4>``, to avoid having to case the integers to double on every use.